apps/snitch_core/priv/repo/migrations/20180314131025_add_unique_index_to_state.exs

```diff
@@ -0,0 +1,8 @@
+defmodule Snitch.Repo.Migrations.AddUniqueIndexToState do
+  use Ecto.Migration
+
+  def change do
+    create unique_index(:snitch_states, [:abbr, :country_id], 
+    message: "(:country_id, :abbr) is already taken")
```

Comment:
Please change this to `{:country_id, :abbr} has already been taken` :sweat_smile: